src/thermo/BinarySolutionTabulatedThermo.cpp

```diff
@@ -161,29 +186,51 @@ void BinarySolutionTabulatedThermo::initThermoXML(XML_Node& phaseNode, const std
             getFloatArray(dataNode, h, true, "J/kmol", "enthalpy");
             getFloatArray(dataNode, s, true, "J/kmol/K", "entropy");
 
+            // Check for partialMolarVolume tag in tabulatedThermo table, take m_speciesMolarVolume[0] if false (for backward compatibility)
```

Comment:
As I mentioned earlier, unless there is a substantial existing userbase for this feature, there's no need to add it to the XML reader.